libs/utils/android/system.py

```diff
@@ -23,15 +23,16 @@
 import pexpect as pe
 
 GET_FRAMESTATS_CMD = 'shell dumpsys gfxinfo {} > {}'
+SYSTRACE_EVENTS_DEFAULT = ['gfx', 'view', 'sched', 'freq', 'idle']
+SYSTRACE_BUFFSIZE_DEFAULT = 10240
 
 class System(object):
     """
     Collection of Android related services
     """
 
     @staticmethod
-    def systrace_start(env, trace_file, time=None,
-                       events=['gfx', 'view', 'sched', 'freq', 'idle']):
+    def systrace_start(env, trace_file, time=None):
```

Comment:
Maybe we can keep backward compatible this API by still passing `events=None`?

If `None` events are specified at the call of this method, then we do the dance to load either the configuration provided ones of the default ones.

Thus priorities will be:
1. call parameter
2. configuration file
3. hard-coded defaults